### PR TITLE
Np 2266 updated doc

### PR DESCRIPTION
**Pull request by svenngjeto**

### CODE REVIEW

**Review by axthosarouris — DISMISSED** [review]

File: template.yaml [hunk 1]
```diff
@@ -149,7 +149,15 @@ Resources:
         openapi: 3.0.3
         info:
           title: NVA Public Search API
-          version: '0.1.0'
+          version: '0.2.0'
+          description: |-
+            Public available search API for resources in NVA. The API is backed by ElasicSearch and uses ElasticSearch query syntax.
```
Comment (brinxmat) [inline template.yaml:154]: Public search API…Elast*T*icSearch and

Comment (brinxmat) [reply template.yaml:154]: Maybe add link to ES-query syntax

**Review by brinxmat — CHANGES_REQUESTED** [review]

File: template.yaml (hunk 1 above)
Comment (svenngjeto) [reply template.yaml]: Fixed typo and added link to AWS supplied documentation

**Review by brinxmat — CHANGES_REQUESTED** [review]

File: template.yaml [hunk 2]
```diff
@@ -149,7 +149,15 @@ Resources:
         openapi: 3.0.3
         info:
           title: NVA Public Search API
-          version: '0.1.0'
+          version: '0.2.0'
+          description: |-
+            Public available search API for resources in NVA. The API is backed by Amazon ElasticSearch and uses [ElasticSearch query syntax](https://opendistro.github.io/for-elasticsearch-docs/docs/elasticsearch/full-text/).
```
[the remark this answers is not in the document]
Comment (svenngjeto) [reply template.yaml]: fixed

File: template.yaml [hunk 3]
```diff
@@ -201,69 +209,166 @@ Resources:
...
-              title:
-                type: string
-              status:
-                type: integer
-              detail:
-                type: string
-              requestId:
-                type: string
-
+                        $ref: "#/components/schemas/Problem"
```
[the remark this answers is not in the document]
Comment (svenngjeto) [reply template.yaml:231]: Not quite sure what you mean; The documentation of an api is in the deployment template and is a result of the devleopers efforts, or lack thereof....  Updating the 'documentation site' is automatic after an initial list of APIs is supplied.

**Review by brinxmat — APPROVED** [review]

**Pull request merged**